hrui/hrui.php

```diff
@@ -934,3 +944,89 @@ function _hrui_menuSetActive($isActive) {
   CRM_Core_DAO::executeQuery("UPDATE civicrm_navigation SET is_active = {$isActive} WHERE name = 'import_custom_fields'");
   CRM_Core_BAO_Navigation::resetNavigation();
 }
+
+/**
+ * Creates Help Menu in navigation bar
+ *
+ * @param Array $menu List of available menu items
+ */
+function _createHelpMenu(&$menu) {
+  $menu[] = array(
+    'attributes' => array(
+      'label'      => 'Help',
```

Comment:
If you look at `_hrui_civix_insert_navigation_menu` I think it provides defaults for "label" and "active", can you try it without them? If it works you can remove them from the other places.